package.json

```diff
@@ -12,6 +12,9 @@
         "lint": "npm run lint:scripts && lerna run lint --stream --concurrency 1",
         "lint:scripts": "eslint --config .eslintrc.js --ext .jsx,.js,.ts,.tsx packages-web/**/src packages-native/**/src",
         "test": "lerna run test --ignore @widgets-resources/utils-react-widgets --stream --concurrency 1",
+        "test:e2e": "lerna run test:e2e --ignore @widgets-resources/utils-react-widgets --stream --concurrency 1",
```

Comment:
Would it be nice to also execute the detox tests when running test:e2e ?

Reply:
Nope, we need to differentiate otherwise it will run web+native even if you dont have changes